lib/tdf3/src/tdf.ts

```diff
@@ -335,6 +336,16 @@ async function getSignature(
   }
 }
 
+function getSignatureHash(signature: Uint8Array, isLegacyTDF: boolean): string {
+  return isLegacyTDF
+    ? base64.encode(hex.encodeArrayBuffer(signature)) //  <= 3.0.0 Legacy format
+    : base64.encodeArrayBuffer(signature); // >= 4.0.0 Current format
+}
+
+function isTargetSpecLegacyTDF(specVersion?: string): boolean {
+  return !!specVersion?.startsWith('3.');
```

Comment:
The `startsWith('3.')` is based on the current definition of the const `isLegacyTDF`
https://github.com/opentdf/web-sdk/blob/a240d66de226f0420486d25c0c904ce2b22bac63/lib/tdf3/src/tdf.ts#L964